recipes/lyra/all/conanfile.py

```diff
@@ -47,13 +47,4 @@ def package(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "lyra")
         self.cpp_info.set_property("cmake_target_name", "bfg::lyra")
-
-        # TODO: to remove in conan v2 once cmake_find_package* generators removed
-        self.cpp_info.components["_lyra"].set_property(
-            "cmake_target_name", "bfg::lyra")
-        self.cpp_info.filenames["cmake_find_package"] = "lyra"
-        self.cpp_info.filenames["cmake_find_package_multi"] = "lyra"
-        self.cpp_info.names["cmake_find_package"] = "bfg"
-        self.cpp_info.names["cmake_find_package_multi"] = "bfg"
-        self.cpp_info.components["_lyra"].names["cmake_find_package"] = "lyra"
-        self.cpp_info.components["_lyra"].names["cmake_find_package_multi"] = "lyra"
+        self.cpp_info.libdirs = []
```

Comment:
As per https://github.com/conan-io/conan-center-index/discussions/17406 we'll take care of removing this bits when the old generators are fully deprecated (Coming very soon!) for now it's ok for them to stay :)


```suggestion
        # TODO: to remove in conan v2 once cmake_find_package* generators removed
        self.cpp_info.components["_lyra"].set_property(
            "cmake_target_name", "bfg::lyra")
        self.cpp_info.filenames["cmake_find_package"] = "lyra"
        self.cpp_info.filenames["cmake_find_package_multi"] = "lyra"
        self.cpp_info.names["cmake_find_package"] = "bfg"
        self.cpp_info.names["cmake_find_package_multi"] = "bfg"
        self.cpp_info.components["_lyra"].names["cmake_find_package"] = "lyra"
        self.cpp_info.components["_lyra"].names["cmake_find_package_multi"] = "lyra"
        self.cpp_info.libdirs = []
```